Gemfile

```diff
@@ -245,6 +245,7 @@ unless ENV["APPLIANCE"]
     gem "capybara",         "~>2.5.0",  :require => false
     gem "coveralls",                    :require => false
     gem "factory_bot",      "~>4.11.1", :require => false
+    gem "faker",            "~>1.8",    :require => false
```

Comment:
Yay, you made this a dependency in our `Gemfile`.  Saw the PR description fly by in my email earlier, but didn't see the code changes, and was very happy you were already head of me when I looked at the diff.

It is cool we aren't adding another dependency with this, but better that we aren't relying on a third party to provide that dependency for us indirectly here.  👍